### ISSUE TITLE
Add PrimaryReadReplicaConnetion again in v2 (or offer alternative)

### ISSUE THREAD

**Issue by AnnaDamm** [facile-it/doctrine-mysql-come-back#67]
Hi!

I would appreciate it, if you could add the PrimaryReadReplicaConnection again.
The upgrade file for 2.0 says that we should use the PrimaryReadReplicaConnection from doctrine/dbal. But since the wrapper class must be the. one from this repository, it cannot work together, as i can only define one connection wrapper class.

This was my setup in 1.9:
```php
[
    'driver_class' => \Facile\DoctrineMySQLComeBack\Doctrine\DBAL\Driver\PDOMySql\Driver::class, // This is where the come back was located before
    'wrapper_class' => \Doctrine\DBAL\Connections\PrimaryReadReplicaConnection::class,
]
```

The setup in 2.0 now should look something like this (due to removed driver classes):
```php
[
    'driver_class' => \Doctrine\DBAL\Driver\PDO\MySQL\Driver::class,
    'wrapper_class' => \Doctrine\DBAL\Connections\PrimaryReadReplicaConnection::class,  // this _should_ be where the come back can be located now?
]
```

Would you give me an example on how to achieve this without the driver class, or add the PrimaryReadReplicaConnetion class again?

**Comment by Jean85** [comment]
There's #58 in progress to revert that change, but unfortunately I wasn't able to complete it.

**Comment by Jean85** [comment]
I've re-added it in #69 and released it as [2.0.0-BETA2](https://github.com/facile-it/doctrine-mysql-come-back/releases/tag/2.0.0-BETA2)
